packages/app-bridge/src/types/Emitter.ts

```diff
@@ -39,6 +40,7 @@ export type EmitterEvents = {
         portalId: number;
         themeAssets: Record<string, Asset[]>;
         prevThemeAssets: Record<string, Asset[]>;
+        template: ThemeTemplate;
```

Comment:
Theme assets should be applied regardless of the template, why adding the info?

Reply:
Because now ThemeSettings are template dependent.
We have a set of ThemeSettings per template, not a single set of theme settings.

Reply:
"ThemeSettings are template dependent"? 🤔 Shouldn't it be the other way around? Shouldn't template depend on themes?

Reply:
@henser let's see if I can explain myself better:
Themes have `default theme settings` (for each template). Then within the guideline the editor can overwrite these `default theme settings` for each `template` those are the `Template Theme Settings`, these sets of settings are per template. These settings are apply when the editor resets or creates a new item, and they can be overwritten for that specific item.